types/react-messenger-checkbox/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for react-messenger-checkbox 0.1.0
```

Comment:
I think this will have to be a `major.minor`: `0.1`
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library